frontend/src/helpers/clusterSettings.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Cluster settings
+ * This file contains functions to store and load cluster settings in local storage.
+ */
+export interface ClusterSettings {
+  /** Optional default namespace to be used */
+  defaultNamespace?: string;
+  /** Only allow namespaces in this list to be selected */
+  allowedNamespaces?: string[];
+  currentName?: string;
```

Comment:
Can you please add for this also? Thanks

Reply:
done